registry/Cargo.toml

```diff
@@ -4,18 +4,47 @@ version = "0.1.0"
 edition = "2021"
 description = "Registry for buffrs, a modern protocol buffer package manager"
 license = "Apache-2.0"
+default-run = "buffrs-registry"
 
 [dependencies]
-buffrs = { path = "../", version = "0.6.4" }
-prost = "0.12.1"
-tonic = "0.10.2"
+async-trait = "0.1.74"
+aws-config = { version = "0.56.1", optional = true }
+aws-sdk-s3 = { version = "0.34.0", optional = true }
+axum_tonic = "0.1.0"
+buffrs = { path = "../", version = "0.7.1" }
+bytes = "1.5.0"
+chrono = "0.4.31"
 clap = { version = "4.3", features = ["cargo", "derive", "env"] }
+eyre = "0.6.8"
+miette = "5.10.0"
+proptest = "1.3.1"
+prost = "0.12.1"
+semver = "1.0.20"
+sqlx = { version = "0.7.2", features = ["runtime-tokio", "postgres", "macros", "migrate"] }
+ssh-key = { version = "0.6.2", features = ["ed25519"] }
+test-strategy = "0.3.1"
+thiserror = "1.0.49"
 tokio = { version = "1.33.0", features = ["full"] }
+tonic = "0.10.2"
 tracing = "0.1.37"
 tracing-subscriber = "0.3.17"
-sqlx = { version = "0.7.2", features = ["runtime-tokio", "postgres", "macros", "migrate"] }
-eyre = "0.6.8"
 url = "2.4.1"
 
 [build-dependencies]
-buffrs = { path = "../", version = "0.7.0" }
+buffrs = { path = "../", version = "0.7.1" }
+
+[features]
+default = ["storage-s3"]
+storage-s3 = ["dep:aws-config", "dep:aws-sdk-s3"]
+test-database = []
```

Comment:
Why do we have this? it doesnt seem good to publish this feature as it seems like a local dev artifact

Reply:
You are right — will remove this.